### ISSUE TITLE
Implement top right menu on malaria profile

### ISSUE THREAD

**Issue by msschroeder** [OpenSRP/opensrp-client-chw-malaria#19]
![Screenshot_20190509-120514](https://user-images.githubusercontent.com/20777928/57504968-4d436480-72ff-11e9-9b6c-55b0b6a8537a.png)

- [x] Registration info re-opens the person's original family member registration form for editing (already implemented)
- [x] Remove this person opens the remove family member form (already implemented)
- [x] New Follow-up Visit option opens the Follow-up Visit form (#7 ). This should be after Registration Info and Remove this person

**Comment by whoisladleo** [comment]
Waiting on issue #7 to be completed

**Comment by msschroeder** [comment]
QA 5 July Build:
- [x] Text should be "Follow-up Visit" as noted in issue, not "FOLLOWUP Visit" as it appear in screenshot below
![Screenshot_20190705-152831](https://user-images.githubusercontent.com/20777928/60722545-e8406f80-9f39-11e9-98dc-4a515bf1ad7e.png)

**Comment by msschroeder** [comment]
This issue is no longer needed as this form is only accessible for the record the follow up visit button.
